### PR TITLE
Get the latest CLI version from www.pulumi.com

### LINKED ISSUE #38: Update post-install instructions
After installation, we don't make any mention about PATHs. (I thought we used to.) And the "Next Steps" bit is a little confusing, as it gives you three options. We should have one, single, and clear next step.

### PR DESCRIPTION
**Pull request by justinvp**
DO NOT MERGE until the new combined site at https://www.pulumi.com has been deployed.

When we cut over to the new site, the new location will be https://www.pulumi.com/latest-version. We'll have a [redirect in place](https://github.com/pulumi/home/pull/507/files#diff-122e28ab90734fe1d0543e7b551f7342R2) that redirects requests to https://pulumi.io/latest-version to https://www.pulumi.com/latest-version, and our installer script will follow the redirect (via `curl -L` and `Invoke-WebRequest` does this automatically), so everything will continue working even without this change.

However, this change updates the script to get the latest version from https://www.pulumi.com/latest-version directly, avoiding the unnecessary redirect request.

Also, update the post-install instructions to have a single CTA while making changes here.

Fixes #38

### CODE REVIEW

File: dist/install.sh [hunk 1]
```diff
@@ -177,7 +177,4 @@ say_blue "=== Pulumi is now installed! 🍹 ==="
 if [ "$EXTRA_INSTALL_STEP" != "" ]; then
     say_white "${EXTRA_INSTALL_STEP}"
 fi
-say_green "+ If you're new to Pulumi, here are some resources for getting started:"
-say_green "      - Getting Started Guide: https://pulumi.io/quickstart"
-say_green "      - Examples Repo: https://github.com/pulumi/examples"
-say_green "      - Create a New Project: Run 'pulumi new' to create a new project using a template"
+say_green "+ If you're new to Pulumi, get started: https://www.pulumi.com/docs/quickstart"
```
Comment (ellismg) [inline dist/install.sh]: Just checking - It is intentional to provide fewer calls to action here, correct?

Comment (justinvp) [reply dist/install.sh]: Yep, that was the fixing #38 part. Sorry to conflate two changes, but I was already updating one of the lines, so just took care of reducing to a single CTA while making changes here.

Comment (ellismg) [reply dist/install.sh]: No worries - Makes sense!

**Comment by justinvp** [comment]
@ellismg, What's the best way to address the Travis failures?

The problem is:

```
node-pre-gyp WARN Pre-built binaries not found for grpc@1.16.1 and node@12.6.0 (node-v72 ABI, glibc) (falling back to source compile with node-gyp) 
```

And then it fails during source compile. Do we need to specify a specific version of Node in `.travis.yaml`?

**Comment by chrsmith** [comment]
@justinvp looks like we can merge this now.

**Comment by justinvp** [comment]
> @justinvp looks like we can merge this now.

@chrsmith, Yes, we can, modulo unrelated Travis failures.

File: dist/install.sh (hunk 1 above)
Comment (bermudezmt) [reply dist/install.sh]: @justinvp How about just saying "Get started with Pulumi: <link>"?

**Pull request merged**